docs/tasks.rst

```diff
@@ -1,3 +1,5 @@
+Checking the version!
+An update to CumulusCI is available. To install the update, run this command: pip install --upgrade cumulusci
```

Comment:
This is a long-standing minor annoyance that we should track as a bug: rebuilding the task docs ends up accidentally including the "you need to upgrade" notice. The fix might be as simple as making sure the code which outputs this notice calls click.echo with err=True so that it gets output to stderr instead of stdout

Reply:
https://github.com/SFDO-Tooling/CumulusCI/issues/2164